ocrd_tesserocr/ocrd-tool.json

```diff
@@ -13,6 +13,10 @@
           "type": "string",
           "enum": ["page", "region", "line", "word", "glyph"],
           "default": "line"
+        },
+        "language": {
```

Comment:
Perhaps using `model` instead of `language` would fit better. In the past, the model described a language, but recently "scripts" where added, so there exist now `script/Latin` or `script/Fraktur`, for example.

For OCR-D,  I expect that we will finally have several models (perhaps for different scripts or for the different centuries), but all will be for the same language.

Reply:
We might also have some (rare) cases where more than one model should be passed.